### ISSUE TITLE
kubectx A=B should use rename

### ISSUE THREAD

**Issue by ahmetb** [ahmetb/kubectx#13]
Currently we're creating alias of a context by running `kubectx A=B`. 

With the new `kubectl context rename` function we should just use it to rename.

**Comment by jsw** [comment]
Whether you use an alias or a rename, it would be great to be able to see the original unaliased/unrenamed context when running `kubectx` with no args, e.g
```
> kubectx
foo = super_long_context_that_i_want_call_foo
some_other_unaliased_context
```

This gives me confidence I didn't accidentally alias/rename something incorrectly.

**Comment by ahmetb** [comment]
I guess doing a `rename` would prevent us from doing what you just described. Even by just cloning the context (what we're doing today) we still don’t have a good way to store the original context‘s name. Therefore what you just described might be not very feasible. I'm not sure it’s very much necessary either (I don’t know of any tools that give you this experience). In addition to these, I’m not sure how people use the `$ kubectx` output today.

It's probably okay to do breaking changes, but this notation may cause more confusion than it helps.

Another idea would be to actually maintain a `~/.kube/kubectx_aliases/` folder that contain text files (alias names) with the name of the original context. This way:
- we don't actually create a clone context in the original kubeconfig file (arguably cleaner)
- we have to look up into this directory while listing, switching-into contexts all the time and merge the results (also have to think about potential overlapping name conflicts)

It’s a deep rabbit hole, not sure if we should go down that way.
I think a simple rename solves a lot of the long name problems without introducing any. Hope this clarifies what I had in mind.

**Comment by jsw** [comment]
@ahmetb Thanks for the detailed explanation of the various considerations. I agree a simple rename makes sense here.

**Comment by ahmetb** [comment]
Now fixed and tagged as v0.4.0.
